### ISSUE TITLE
Revenue charts have too high precision

### ISSUE THREAD

**Issue by winsmith** [TelemetryDeck/Viewer#451]
Charts in the Revenue Section can show many decimals, which is un-intuitive because people expect two decimals for currency. Might depend on #450

**Comment by Jeehut** [comment]
Possible duplicate of https://github.com/TelemetryDeck/Issues/issues/416?
